configure

```diff
@@ -270,7 +270,7 @@ var_add GLIB_HEADERS "$(pkg-config --cflags glib-2.0)"
 var_add GLIB_LIBS "$(pkg-config --libs glib-2.0)"
 var_add PG_MARCH "core-avx-i"
 var_add PREFIX "/usr/local/"
-var_add EXTRA_CFLAGS ""
+var_add EXTRA_CFLAGS "-g -O2"
```

Comment:
The flag -g is for debugging purpose right?  Maybe we should remove it... (I don't now about the -02...)

Reply:
maybe,
I've use that as default flag because that the default flag on autotool
also, I like debug symbols by default, size doesn't matter that much, debug-ability does